main/src/main/java/org/mobilitydata/gtfsvalidator/validator/LocationHasStopTimesValidator.java

```diff
@@ -80,6 +80,8 @@ public void validate(NoticeContainer noticeContainer) {
 
   /**
    * Describes a location in {@code stops.txt} that is not a stop but has a stop time associated.
+   *
+   * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
Excellent, this is already an error at Google.

Reply:
As suggested, this PR has been split into different PRs: one for each notice (see #1021). As of today, we do not have the statistics yet for GTFS datasets, but will update when we have them.